source/reference/command/replSetGetStatus.txt

```diff
@@ -195,23 +195,13 @@ following fields:
 
    .. versionadded:: 3.4
 
-   The ``optimes`` field holds a document that contains information on
-   various optimes. Optimes are a reference to a position in the
-   replication "oplog", and are reported here to show replication
-   progress for various operations.
-   
-   If using :rsconf:`protocolVersion: 1 <protocolVersion>`, the optime
-   value is a document that contains:
-
-   - ``ts``, the :ref:`Timestamp <document-bson-type-timestamp>` of
-     the operation.
+   The ``optimes`` field holds a document that contains
+   :term:`optimes <optime>` showing replication progress for various
+   types of operations.
```

Comment:
As far as I can see, there are three main differences:

- s/information on various optimes/optimes
- s/reported here to show/showing
- No longer notes different behavior with protocol version 0

The first is only incorrect if the following documentation is also wrong: the documented document fields are just optimes.